test/bin/ci_phase_iso_build.sh

```
@@ -182,7 +182,8 @@ cd "${ROOTDIR}/test/"
 source "${SCRIPTDIR}/common.sh"
 
 # shellcheck source=test/bin/common_versions_verify.sh
-source "${SCRIPTDIR}/common_versions_verify.sh"
+# source "${SCRIPTDIR}/common_versions_verify.sh"
+source "${SCRIPTDIR}/manage_common_versions.sh" verify
```

Shellcheck (0.11.0)

[info] Not following: test/bin/common_versions_verify.sh was not specified as input (see shellcheck -x).

(SC1091)